src/main/features/response/routes/full-admission/payment-plan.ts

```diff
@@ -1,26 +1,45 @@
 import * as express from 'express'
 
+import * as _ from 'lodash'
+
 import { Paths, FullAdmissionPaths } from 'response/paths'
 
 import { ErrorHandling } from 'shared/errorHandling'
 import { Form } from 'forms/form'
 import { DraftService } from 'services/draftService'
 import { User } from 'idam/user'
-import { PaidAmount } from 'ccj/form/models/paidAmount'
 import { DefendantPaymentPlan } from 'response/form/models/defendantPaymentPlan'
 import { FormValidator } from 'forms/validation/formValidator'
 import { FeatureToggleGuard } from 'guards/featureToggleGuard'
 import { ResponseDraft } from 'response/draft/responseDraft'
 import { Draft } from '@hmcts/draft-store-client'
 import { Claim } from 'claims/models/claim'
+import { createPaymentPlan } from 'common/paymentPlan'
+import { PaymentSchedule } from 'features/ccj/form/models/paymentSchedule'
+
+function mapFrequencyInWeeks (frequency: PaymentSchedule): number {
+  switch (frequency) {
+    case PaymentSchedule.EACH_WEEK:
+      return 1
+    case PaymentSchedule.EVERY_TWO_WEEKS:
+      return 2
+    case PaymentSchedule.EVERY_MONTH:
+      return 4
+    default:
+      return undefined
+  }
+}
 
-function renderView (form: Form<PaidAmount>, res: express.Response): void {
+function renderView (form: Form<DefendantPaymentPlan>, res: express.Response): void {
   const claim: Claim = res.locals.claim
   const draft: Draft<ResponseDraft> = res.locals.responseDraft
   const alreadyPaid: number = draft.document.paidAmount.amount || 0
```

Comment:
I cannot recall this in new prototype, we should remove it. I think `draft.document.paidAmount` is never instantiated.

Reply:
its not in new prototype, i removed it.